src/redux/epics/team.js

```diff
@@ -0,0 +1,33 @@
+import { find, propEq, equals } from 'ramda'
+import { combineEpics, ofType } from "redux-observable"
+import { filter, map, mergeMap, withLatestFrom } from 'rxjs/operators'
+import { AppActionCreator, TEAM_ACTIONS, TeamActionCreator } from '../actions'
+import teamService from "../../service/team"
+import { getTeamsCollection } from '../selectors'
+import { of, concat } from 'rxjs'
+
```

Comment:
Because you ar adding new technologies like redux, redux observable and rxjs, start documenting the why of this libraries inside the readme, so the other developers can understand!

Reply:
There is already an issue for this. The Acceptance Criteria have been updated. https://github.com/Phlammariont/happiny/issues/20